src/app/(tabs)/map.tsx

```diff
@@ -144,6 +110,68 @@ export const MapScreen = (): JSX.Element => {
 
     const mapRef = useRef<WebView>(null)
     const router = useRouter()
+    const navigation = useNavigation()
+    const { t } = useTranslation('common')
+
+    useEffect(() => {
+        navigation.setOptions({
+            headerRight: () => (
+                <Pressable
+                    onPress={() => {
+                        router.push('(map)/advanced')
+                    }}
+                >
+                    <Ionicons
+                        name="options-outline"
+                        size={24}
+                        color={colors.text}
+                    />
+                </Pressable>
+            ),
+            headerSearchBarOptions: {
+                placeholder: t('pages.map.search'),
```

Comment:
If you open the search bar on android and then click on a Room that the Popup opens, the advanced search icon is shown left to the search as shown in the screenshot:

![image](https://github.com/neuland-ingolstadt/neuland.app-native/assets/39560569/81237e0f-4936-41be-b9c8-0469ed6b5fca)